packages/paste-website/src/pages/components/combobox/index.mdx

```diff
@@ -184,7 +215,23 @@ In the example below we are checking the `groupName` and rendering an icon next
   {groupedLabelComboboxExample}
 </LivePreview>
 
-#### Controlled Combobox
+### Combobox using Option Template
+
+Use the option template to display more complex options in a listbox.
+
+The `optionTemplate` prop allows you to pass `jsx` to customize the options. Note that we use native HTML input elements
+to build Combobox and these HTML elements don't allow for images, icons, or svgs to be added even with the option template.
+
+<LivePreview
+  scope={{Combobox, LinkExternalIcon, MediaBody, MediaObject, MediaFigure, Text}}
+  noInline
+  language="jsx"
+  showOverflow
+>
+  {optionTemplateExample}
```

Comment:
Since having links/anchors as combobox options is kinda jank, can you remove the ExternalLinkIcon from the optionTemplateExample?

It can look something like this: https://docs.google.com/document/d/143idKgnfqFCmSceZZ67nIcLCxRnBsuxYPB8TMxTxKno/edit#bookmark=id.aa46w1hfargd